src/Select/stories/index.stories.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react'
+import { Story, ComponentMeta } from '@storybook/react'
+import { Form, FormItemProps } from 'antd'
+import { MagnifyingGlass } from 'phosphor-react'
+import '../../assets/style/form-item.css'
```

Comment:
Please separate css import from component import. css import should be after component import with a line gap between them